meilisearch/landing
# Add values block

# Pull Request

## Related issue
Closes #480

## What does this PR do?
- Addition of a new block `Our values`
- Addition of the corresponding story in storybook

## How to test

Go to [the deploy branch](https://landing-git-addvaluesblock-meili.vercel.app/about-us) and check the `Our values` block
You can also `yarn storybook` and go to [the corresponding story](http://localhost:6006/?path=/story/blocks-values--default).

## PR checklist
Please check if your PR fulfills the following requirements:

Thank you so much for contributing to Meilisearch!
